decoder/DecoderApi_unittest.cpp

```diff
@@ -95,13 +116,15 @@ class TestDecodeFrames {
         const char* mime)
         : m_data(data)
         , m_mime(mime)
+        , m_timestamp(0)
```

Comment:
Please do not put time stamp here, the time stamp is just for the flush case, you may need put the timestap at flush case only